superset-frontend/src/components/CronPicker/CronPicker.tsx

```diff
@@ -110,22 +110,86 @@ export const CronPicker = styled((props: CronProps) => (
     <ReactCronPicker locale={LOCALE} {...props} />
   </ConfigProvider>
 ))`
-  .react-js-cron-field {
-    margin-bottom: 0px;
-  }
-  .react-js-cron-select:not(.react-js-cron-custom-select) > div:first-of-type,
-  .react-js-cron-custom-select {
-    border-radius: ${({ theme }) => theme.gridUnit}px;
-    background-color: ${({ theme }) =>
-      theme.colors.grayscale.light4} !important;
-  }
-  .react-js-cron-custom-select > div:first-of-type {
-    border-radius: ${({ theme }) => theme.gridUnit}px;
-  }
-  .react-js-cron-custom-select .ant-select-selection-placeholder {
-    flex: auto;
-  }
-  .react-js-cron-custom-select .ant-select-selection-overflow-item {
-    align-self: center;
-  }
+  ${({ theme }) => css`
```

Comment:
Need to look at this styling, the labels are now on separate lines from the inputs without making any changes since my last correction

Reply:
Seems the basic styling from `react-js-cron/dist/styles.css` went missing from my local env. Not sure if it will be reproduced remotely